rskj-core/src/main/java/co/rsk/peg/pegin/PeginProcessAction.java

```diff
@@ -3,5 +3,5 @@
 public enum PeginProcessAction {
     CAN_BE_REGISTERED,
     CAN_BE_REFUNDED,
-    CANNOT_BE_PROCESSED
+    CANNOT_BE_REFUNDED
```

Comment:
Just an idea, to rename this for clarity

```
REGISTER,
REFUND,
NO_REFUND
```
What do you think? A bit more straightforward. Last one could also be `IGNORE` perhaps, not sure

Reply:
Looks simpler. I'll update it.